courses/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+
+# Create your models here.
+
+class Languages(models.Model):
+    language_name = models.CharField(max_length=20)
+
+    def __str__(self):
+        return self.language_name
```

Comment:
Recommended would be string interpolation as best practice `"{0} - {1}".format(self.language_name, self.slug)`

Reply:
read up on this methods advantages